FirebaseAuth/Sources/Auth/FIRAuth.m

```diff
@@ -357,7 +358,8 @@ @interface FIRAuth () <FIRLibrary, FIRComponentLifecycleMaintainer>
 @property(nonatomic, strong, nullable) FIRAuthStoredUserManager *storedUserManager;
 
 /** @fn initWithApp:
-    @brief Creates a @c FIRAuth instance associated with the provided @c FIRApp instance.
+    @brief Creates a @c FIRAuth instance associated with the provided @c FIRApp instance and
+   AppCheck Interop.
```

Comment:
nit: You can revert the change to this comment, now that we're no longer passing the App Check interop into the initializer.

Reply:
Sure, thx, solved.